browser/resources/settings/brave_wallet_page/add_wallet_network_dialog.html

```diff
@@ -37,13 +37,13 @@
   <div slot="body">
     <template is="dom-if" if="[[addNewAllowed]]">
       <div class="search-separator">
-        <cr-searchable-drop-down items="[[searchItems_]]"
+        <searchable-drop-down-cros items="[[searchItems_]]"
```

Comment:
why are we using a chromium os file here?

Reply:
I think we just want to drop `cros` as it makes no sense here

Reply:
Fixed in 2c31c21eb86c5ac254fbf15f6bd375e7ea7c2423.